implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -152,11 +152,13 @@ impl<'a> Rpc<'a> {
             .ctx
             .send_and_receive(route.clone(), req.to_vec()?)
             .await
-            .context("Failed to receive response from node")?;
+            .map_err(|_err| {
+                // Overwrite error to swallow inner cause and hide it from end-user
+                anyhow!("The request timed out, please try again")
```

Comment:
Is a timeout the only possible error here? `_err` does not bring any useful information? Should it still be logged and not just dropped?

Reply:
Since it's a generic code path shouldn't we avoid the `please try again` part?

Reply:
> Is a timeout the only possible error here?

Yes, since the NodeManager [will return a response with the error data](https://github.com/build-trust/ockam/blob/f0a16e18a77b60e0f6c1b993db699e44c478c40c/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L757
https://github.com/phillyphil91/ockam/blob/f0a16e18a77b60e0f6c1b993db699e44c478c40c/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L757).

> _err does not bring any useful information?

I think not in this context. It contains low-level information of what went wrong. I'm not sure how to translate that information into meaningful data for the command user. I guess that, in case the user wants to know the technical detail of what went wrong, they should look at the logs?

> Should it still be logged and not just dropped?

It is [logged in the NodeManager](https://github.com/build-trust/ockam/blob/f0a16e18a77b60e0f6c1b993db699e44c478c40c/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L746
https://github.com/phillyphil91/ockam/blob/f0a16e18a77b60e0f6c1b993db699e44c478c40c/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L746).

Logging the error on the ockam_command would still be useful if the user runs the command with logs enabled, so they don't have to go to the node's logs. Do you think is worth it?

Reply:
>  I guess that, in case the user wants to know the technical detail of what went wrong, they should look at the logs?
> Do you think is worth it?

No I think it's ok as long as the log message exists somewhere to do the diagnostic.

Reply:
> Since it's a generic code path shouldn't we avoid the please try again part?

@davide-baldo do you think "The request timed out" would be enough? Or would you add a generic help text as to what to do next?

Reply:
Maybe something along the lines "check your parameters or try again" to highlight the possibility that the timeout might not the root issue

Reply:
I've changed a little the message.